samples/get_object.bal

```diff
@@ -16,14 +17,11 @@ s3:ClientConfiguration amazonS3Config = {
 s3:Client amazonS3Client = check new (amazonS3Config);
 
 public function main() returns error? {
-    var getObjectResponse = amazonS3Client->getObject(bucketName, "test.txt");
-    if (getObjectResponse is s3:S3Object) {
-        log:printInfo(getObjectResponse.toString());
-        byte[]? byteArray = getObjectResponse["content"];
-        if (byteArray is byte[]) {
-            string content = check strings:fromBytes(byteArray);
-            log:printInfo("Object content: " + content);
-        }
+    stream<byte[], io:Error?>|error getObjectResponse = amazonS3Client->getObject(bucketName, "test.txt");
```

Comment:
As we have introduced a new parameter `byteArraySize `,
Shall we add a sample to show how to use it?

Reply:
`byteArraySize` is an optional value and it is not so important. Plugin can suggest this optional parameter to the user.